run_tests.py.in

```diff
@@ -301,12 +280,20 @@ def clean_test_tree(path):  # type: (str) -> None
 def filter_tests(path):  # type: (str) -> list
     """Filter and sort tests in ascending order."""
     test_files = [str(f) for f in Path(path).rglob('*.re')]
-    test_files.sort()
```

Comment:
Bash version of test runner had this option. Don't we need it anymore?

Reply:
It doesn't make sense to sort them since we use `imap_unordered` anyway, does it?